core/application/app_config.hpp

```diff
@@ -66,6 +66,11 @@ namespace kagome::application {
      * @return true if node in only finalizing mode, otherwise false.
      */
     virtual bool is_only_finalizing() const = 0;
+
+    /**
+     * @return true if need to force block production
```

Comment:
I think that it might need clarification why if we already synchronized it requires to force block production, because it requires some delving into context to understand it.

Reply:
agree